PIMS/Controllers/ProductsController.cs

```diff
@@ -0,0 +1,152 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Services.Helper;
+using ServicesContract.Model;
+using ServicesContracts;
+using ServicesContracts.Model;
+using System;
+using System.Threading.Tasks;
+
+namespace Assignment_1.Controllers
+{
+    [Route("/products")]
+    public class ProductsController : Controller
+    {
+        private readonly IProducts _products;
+        private readonly ExportExcel _exportExcel;
+
+        public ProductsController(IProducts products, ExportExcel exportExcel)
+        {
+            _products = products;
+            _exportExcel = exportExcel;
+        }
+
+        [HttpGet("/")]
```

Comment:
Simplified Route Definitions to Maintain Consistency
Why: Simplifying and standardizing routes makes them easier to read, understand, and maintain. Consistent route definitions also help in avoiding errors and confusion

 The leading slash / was removed from the Route attribute and the action method route. This ensures consistency in route definitions.

Reply:
Resolved